packages/terra-grid/src/Grid.scss

```diff
@@ -4,44 +4,47 @@
 
 // Grid Component
 // ========================================================
+/* stylelint-disable selector-class-pattern */
+:local {
+  .grid {
+    box-sizing: border-box;
+    display: flex;
+    flex-basis: auto;
+    flex-direction: row;
+    flex-grow: 0;
+    flex-shrink: 1;
+    flex-wrap: wrap;
+    margin-left: calc(-1*var(--terra-Grid-gutter-width));
```

Comment:
This isn't supported in IE10 so we need to ensure there is a postCSS plugin converting this to a discrete value.